service_container/lazy_services.rst

```diff
@@ -84,15 +84,18 @@ To check if your lazy service works you can check the interface of the received
     // the output should include "Symfony\Component\VarExporter\LazyObjectInterface"
 
 You can also configure your service's laziness thanks to the
-:class:`Symfony\\Component\\DependencyInjection\\Attribute\\Autoconfigure` attribute.
+:class:`Symfony\\Component\\DependencyInjection\\Attribute\\Autoconfigure` or the
+:class:`Symfony\\Component\\DependencyInjection\\Attribute\\Lazy` attribute.
 For example, to define your service as lazy use the following::
 
     namespace App\Twig;
 
     use Symfony\Component\DependencyInjection\Attribute\Autoconfigure;
+    // use Symfony\Component\DependencyInjection\Attribute\Lazy;
     use Twig\Extension\ExtensionInterface;
 
     #[Autoconfigure(lazy: true)]
+    // #[Lazy]
```

Comment:
I think this can be confusing. I would suggest keeping the `Autoconfigure` as is and add another paragraph with something like "Another possibility is to use the `#[Lazy]` attribute etc..."

Reply:
And a `versionadded:: 7.1` directive would be added at the end of the second paragraph 👍

Reply:
Sorry for the delay, fixed!